File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/dto/d/ImplVtDTO.java
package iih.ci.ord.dto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 实体 DTO数据 
 * 
 */
public class ImplVtDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 *  主键
	 * @return String
	 */
	public String getId_implvt() {
		return ((String) getAttrVal("Id_implvt"));
	}
	/**
	 *  主键
	 * @param Id_implvt
	 */
	public void setId_implvt(String Id_implvt) {
		setAttrVal("Id_implvt", Id_implvt);
	}
	/**
	 * 所属集团id
	 * @return String
	 */
	public String getId_grp() {
		return ((String) getAttrVal("Id_grp"));
	}
	/**
	 * 所属集团id
	 * @param Id_grp
	 */
	public void setId_grp(String Id_grp) {
		setAttrVal("Id_grp", Id_grp);
	}
	/**
	 * 所属集团名称
	 * @return String
	 */
	public String getName_grp() {
		return ((String) getAttrVal("Name_grp"));
	}
	/**
	 * 所属集团名称
	 * @param Name_grp
	 */
	public void setName_grp(String Name_grp) {
		setAttrVal("Name_grp", Name_grp);
	}
	/**
	 * 所属组织id
	 * @return String
	 */
	public String getId_org() {
		return ((String) getAttrVal("Id_org"));
	}
	/**
	 * 所属组织id
	 * @param Id_org
	 */
	public void setId_org(String Id_org) {
		setAttrVal("Id_org", Id_org);
	}
	/**
	 * 所属组织名称
	 * @return String
	 */
	public String getName_org() {
		return ((String) getAttrVal("Name_org"));
	}
	/**
	 * 所属组织名称
	 * @param Name_org
	 */
	public void setName_org(String Name_org) {
		setAttrVal("Name_org", Name_org);
	}
	/**
	 * 服务分类id
	 * @return String
	 */
	public String getId_srvca() {
		return ((String) getAttrVal("Id_srvca"));
	}
	/**
	 * 服务分类id
	 * @param Id_srvca
	 */
	public void setId_srvca(String Id_srvca) {
		setAttrVal("Id_srvca", Id_srvca);
	}
	/**
	 * 服务分类内编码
	 * @return String
	 */
	public String getSrvca_innercode() {
		return ((String) getAttrVal("Srvca_innercode"));
	}
	/**
	 * 服务分类内编码
	 * @param Srvca_innercode
	 */
	public void setSrvca_innercode(String Srvca_innercode) {
		setAttrVal("Srvca_innercode", Srvca_innercode);
	}
	/**
	 * 服务分类名称
	 * @return String
	 */
	public String getName_srvca() {
		return ((String) getAttrVal("Name_srvca"));
	}
	/**
	 * 服务分类名称
	 * @param Name_srvca
	 */
	public void setName_srvca(String Name_srvca) {
		setAttrVal("Name_srvca", Name_srvca);
	}
	/**
	 * 需要执行的sql
	 * @return String
	 */
	public String getExec_sql() {
		return ((String) getAttrVal("Exec_sql"));
	}
	/**
	 * 需要执行的sql
	 * @param Exec_sql
	 */
	public void setExec_sql(String Exec_sql) {
		setAttrVal("Exec_sql", Exec_sql);
	}
	/**
	 * base64编码的sql语句
	 * @return String
	 */
	public String getExec_sql_base64() {
		return ((String) getAttrVal("Exec_sql_base64"));
	}
	/**
	 * base64编码的sql语句
	 * @param Exec_sql_base64
	 */
	public void setExec_sql_base64(String Exec_sql_base64) {
		setAttrVal("Exec_sql_base64", Exec_sql_base64);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/d/ImplVtDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.d
{
    /// <summary>
    /// ImplVtDTO 的摘要说明。
    /// </summary>
    public class ImplVtDTO : BaseDTO {

        public ImplVtDTO() {
 
        }

        /// <summary>
        ///  主键
        /// </summary>
		public string Id_implvt {
            get { return getAttrVal<string>("Id_implvt",null); }
            set { setAttrVal<string>("Id_implvt", value); }
        }

        /// <summary>
        /// 所属集团id
        /// </summary>
		public string Id_grp {
            get { return getAttrVal<string>("Id_grp",null); }
            set { setAttrVal<string>("Id_grp", value); }
        }

        /// <summary>
        /// 所属集团名称
        /// </summary>
		public string Name_grp {
            get { return getAttrVal<string>("Name_grp",null); }
            set { setAttrVal<string>("Name_grp", value); }
        }

        /// <summary>
        /// 所属组织id
        /// </summary>
		public string Id_org {
            get { return getAttrVal<string>("Id_org",null); }
            set { setAttrVal<string>("Id_org", value); }
        }

        /// <summary>
        /// 所属组织名称
        /// </summary>
		public string Name_org {
            get { return getAttrVal<string>("Name_org",null); }
            set { setAttrVal<string>("Name_org", value); }
        }

        /// <summary>
        /// 服务分类id
        /// </summary>
		public string Id_srvca {
            get { return getAttrVal<string>("Id_srvca",null); }
            set { setAttrVal<string>("Id_srvca", value); }
        }

        /// <summary>
        /// 服务分类内编码
        /// </summary>
		public string Srvca_innercode {
            get { return getAttrVal<string>("Srvca_innercode",null); }
            set { setAttrVal<string>("Srvca_innercode", value); }
        }

        /// <summary>
        /// 服务分类名称
        /// </summary>
		public string Name_srvca {
            get { return getAttrVal<string>("Name_srvca",null); }
            set { setAttrVal<string>("Name_srvca", value); }
        }

        /// <summary>
        /// 需要执行的sql
        /// </summary>
		public string Exec_sql {
            get { return getAttrVal<string>("Exec_sql",null); }
            set { setAttrVal<string>("Exec_sql", value); }
        }

        /// <summary>
        /// base64编码的sql语句
        /// </summary>
		public string Exec_sql_base64 {
            get { return getAttrVal<string>("Exec_sql_base64",null); }
            set { setAttrVal<string>("Exec_sql_base64", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Id_implvt", "Id_grp", "Name_grp", "Id_org", "Name_org", "Id_srvca", "Srvca_innercode", "Name_srvca", "Exec_sql", "Exec_sql_base64"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.dto.d.ImplVtDTO";
        }
    }
}
